### DISCUSSION TITLE
My account was banned?

### DISCUSSION THREAD

**Discussion by Dwhitey11** [chromiecraft/chromiecraft#6221] [General]
Hey I was playing this game with my buddy for a little bit, haven't been on much (I think a level 50 warrior). I toke a break for a couple months and when I came back my account "has been closed". I'm just wondering if I can have some insight. My username on CC is dwhitey1811

**Comment by 55Honey** [comment]
Hello @Dwhitey11,
thanks for reaching out.
Since discussions on GitHub are visible to anyone, we encourage the use of our [discord](https://discord.gg/chromiecraft) to open tickets only visible to the server's staff.

There is no account on ChromieCraft with the username you've mentioned. Either you're connecting to a different server, or there is a typo or misunderstanding somewhere.

Regards
